docs/getting-started.md

```diff
@@ -133,6 +136,7 @@ As part of creating your Meltano project, we automatically added your first [env
 1. Activate your environment for your shell session:
 
     ```bash
```

Comment:
Instead of having a comment in the `bash` block for Windows, we should use a tabbed code block with a tab for `bash`, and another for `powershell`. Unfortunately I don't believe Kramdown supports this. For now we can just use two code blocks instead, one after the other.

@afolson when we get started on https://github.com/meltano/meltano/issues/5948, we should consider switching to a different markdown processor. I'm not sure how much attachment there is to our use of Jekyll for the docs. If there isn't much, and it wouldn't be too much trouble, we could switch the entire docs site to Mkdocs, and get the more powerful/extensible markdown process for free. Otherwise we can have the generated API docs as a sub-site under the Jekyll docs site, and make an effort towards making their styles match at least enough that switching between them wouldn't be distracting.

Reply:
Regarding use of "tabbed code block". My company does this and they can be extremely helpful in providing clarity. However, we are still working on exactly how/when to use them for the best user experience. Here is some recent advice I gave to my content team that you may want to consider:
 
 - **No tabs where possible** - if the command is the same between bash and powershell, and it's not so long that you need line continuation characters just show one code block. Less cognitive load for the reader, less maintenance.
 
 - **Link the tabs** - if you have multiple tabbed code block examples in a doc, when you switch one to bash or powershell the rest should switch as well. Prevents reader from 

 - **Line continuation characters** - \ for bash and ` for powershell. Use them sparingly as they will force you to have different tabs for bash/powershell.

 - **Variables** - Variables can make for easier to read commands, but they will force you to have different tabs and individual lines that use variables cannot be copied and run on their own.

Reply:
@seajhawk Thanks for the advice around tabbed code blocks. Since we can't use them yet, we should split this into 2 code blocks, one after the other.

Reply:
@WillDaSilva - split now.